common/src/main/java/org/conscrypt/NativeCrypto.java

```diff
@@ -925,55 +928,50 @@ static native void SSL_set_client_CA_list(long ssl, NativeSsl ssl_holder, byte[]
     };
 
     /** Protocols to enable by default when "TLSv1.1" is requested. */
-    static final String[] TLSV11_PROTOCOLS = new String[] {
-            SUPPORTED_PROTOCOL_TLSV1,
-            SUPPORTED_PROTOCOL_TLSV1_1,
-            SUPPORTED_PROTOCOL_TLSV1_2,
-    };
+    static final String[] TLSV11_PROTOCOLS = TLSV12_PROTOCOLS;
 
     /** Protocols to enable by default when "TLSv1" is requested. */
-    static final String[] TLSV1_PROTOCOLS = new String[] {
-            SUPPORTED_PROTOCOL_TLSV1,
-            SUPPORTED_PROTOCOL_TLSV1_1,
-            SUPPORTED_PROTOCOL_TLSV1_2,
-    };
+    static final String[] TLSV1_PROTOCOLS = TLSV11_PROTOCOLS;
 
     static final String[] DEFAULT_PROTOCOLS = TLSV12_PROTOCOLS;
+    private static final String[] SUPPORTED_PROTOCOLS = DEFAULT_PROTOCOLS;
 
     static String[] getSupportedProtocols() {
-        return TLSV12_PROTOCOLS.clone();
+        return SUPPORTED_PROTOCOLS.clone();
     }
 
     static void setEnabledProtocols(long ssl, NativeSsl ssl_holder, String[] protocols) {
         checkEnabledProtocols(protocols);
```

Comment:
Maybe worth a comment here like:

// Although the Java API gives a list of protocols, TLS wants a contiguous range. We pick the lowest contiguous [at least that's what it looks like] range the caller specified.

Reply:
Done.